x509/src/certificate.rs

```diff
@@ -21,9 +24,65 @@ pub fn default_zero() -> u32 {
     0
 }
 
-/// only support v3 certificates
+/// only supporting v3 certificates, but all versions are listed here
 /// Version  ::=  INTEGER  {  v1(0), v2(1), v3(2)  }
-pub const X509_CERT_VERSION: u8 = 2;
+
+/// Version identifier for X.509 certificates. In practice, only v3 is used.
+#[derive(Clone, Debug, Copy, PartialEq, Eq, PartialOrd)]
```

Comment:
```suggestion
#[derive(Copy, Clone, Debug, Copy, PartialEq, Eq, PartialOrd)]
```

Reply:
Copy was there. Reordered so it is now first.